RxSwiftExtensions/RxCocoa/RxCocoa_Binder.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  RxCocoa_Binder.swift
+//  RxSwiftExtensions
+//
+//  Created by 윤중현 on 22/11/2018.
+//  Copyright © 2018 tokijh. All rights reserved.
+//
+
+import RxSwift
+import RxCocoa
+
+extension ObservableType {
+    public func bind<O: ObserverType>(to observers: [O]) -> Disposable where O.E == E {
+        let shared = self.share()
+        let disposables = observers.map({ shared.bind(to: $0) })
```

Comment:
How about this? 👻 
```swift
let disposables = observers.map { shared.bind(to: $0) }
```

Reply:
Not bad. But I prefer to use `({ })` for things that can be expressed in one line.
More than two lines, use `{ }` What do you think?

Reply:
@tokijh It seems to be possible to express in one line

Reply:
@cruisediary I don't care. Please make a PR If you want to fix it 👍